src/app/src/reducers/FiltersReducer.js

```diff
@@ -71,7 +71,12 @@ export default createReducer({
     [updatePPEFilter]: (state, payload) => update(state, {
         ppe: { $set: payload },
     }),
-    [resetAllFilters]: () => initialState,
+    [resetAllFilters]: (state, payload) => update(initialState, {
```

Comment:
Although it is conventional to call the second argument `payload` I think we should consider using `isEmbedded` in this case. It is more self documenting and I can argue that since we are not putting the payload into the state, just using it to change behavior, that the break with convention is justified.